app.js

```diff
@@ -0,0 +1,37 @@
+const fs = require('fs').promises;
+
+require('console-inject');
+const arg = require('arg');
+const exportDiagram = require('./index');
+
+// Options are modelled after the Draw.io Desktop CLI; ideally we should be a
+// 1:1 substitute.
+const args = arg({
```

Comment:
We'll need to improve error handling when arguments aren't set, and provide some useful help.